hack/install-ocs-operator.sh

```diff
@@ -32,40 +32,16 @@ data:
   ROOK_CSI_DISABLE_DRIVER: "false"
 EOF
 
-
-patch_ocs_client_operator_config_configmap() {
-    while true; do
-        if oc get cm ocs-client-operator-config -n openshift-storage; then
-            oc patch cm ocs-client-operator-config -n openshift-storage --type merge -p '{"data":{"DEPLOY_CSI":"false"}}'
-            sleep 2
-        fi
-    done
-}
-
-
 "$OPERATOR_SDK" run bundle "$ROOK_BUNDLE_FULL_IMAGE_NAME" --timeout=10m --security-context-config restricted -n "$INSTALL_NAMESPACE"
 "$OPERATOR_SDK" run bundle "$CSI_ADDONS_BUNDLE_FULL_IMAGE_NAME" --timeout=10m --security-context-config restricted -n "$INSTALL_NAMESPACE"
```

Comment:
We don't even need CSI addons, It was being deployed because of the OCS-client-operator.

Reply:
done.

Reply:
done.